src/test/java/seedu/address/testutil/TypicalAttendances.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.testutil;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.model.attendance.Attendance;
+/**
+ * A utility class containing a list of {@code Attendance} objects to be used in tests.
+ */
+public class TypicalAttendances {
+
+    public static final Attendance ALICE_AURORA = new AttendanceBuilder().withMemberName("Alice Pauline")
+            .withEventName("Aurora Borealis").withHours("3").withRemark("Role: Photographer").build();
+
+    public static final Attendance BENSON_BOXING = new AttendanceBuilder().withMemberName("Benson Meier")
+            .withEventName("Boxing Day").withHours("7").withRemark("Bring boxing gloves").build();
+
+    public static final Attendance CARL_CNY = new AttendanceBuilder().withMemberName("Carl Kurz")
+            .withEventName("Chinese New Year").withHours("3").withRemark("Brought red packets").build();
+    public static final Attendance DANIEL_DOG_DAY = new AttendanceBuilder().withMemberName("Daniel Meier")
+            .withEventName("Dog Celebration Day").withHours("5").withRemark("Bringing dogs").build();
+
+    public static final Attendance ELLE_EHCO_DAY = new AttendanceBuilder().withMemberName("Elle Meyer")
```

Comment:
```suggestion
    public static final Attendance ELLE_ECHO_DAY = new AttendanceBuilder().withMemberName("Elle Meyer")
```
Just a minor typo here.